system_tests/features/steps/get_info_endpoints.py

```diff
@@ -0,0 +1,200 @@
+
+
+@given('the system requests frontstage endpoint info')
+def requests_frontstage_endpoint_info(context):
+    context.response = requests.get(Config.FRONTSTAGE_SERVICE + Config.INFO)
+
+
+@given('the system requests iac endpoint info')
+def requests_iac_endpoint_info(context):
+    context.response = requests.get(Config.IAC_SERVICE + Config.INFO)
+
+
+@given('the system requests notify gateway endpoint info')
+def requests_notify_gateway_endpoint_info(context):
+    context.response = requests.get(Config.NOTIFY_GATEWAY_SERVICE + Config.INFO)
+
+
+@given('the system requests party endpoint info')
+def requests_party_endpoint_info(context):
+    context.response = requests.get(Config.PARTY_SERVICE + Config.INFO)
+
+
+@given('the system requests response operations ui endpoint info')
+def requests_response_operations_ui_endpoint_info(context):
+    context.response = requests.get(Config.RESPONSE_OPERATIONS_UI + Config.INFO)
+
+
+@given('the system requests sample endpoint info')
+def requests_sample_endpoint_info(context):
+    context.response = requests.get(Config.SAMPLE_SERVICE + Config.INFO)
+
+
+@given('the system requests secure message endpoint info')
+def requests_secure_message_endpoint_info(context):
+    context.response = requests.get(Config.SECURE_MESSAGE_SERVICE + Config.INFO)
+
+
+@given('the system requests survey endpoint info')
+def requests_survey_endpoint_info(context):
+    context.response = requests.get(Config.SURVEY_SERVICE + Config.INFO)
+
+
+@when('the action endpoint info is returned')
+def action_endpoint_info_is_returned(context):
+    response = json.loads(context.response.text)
+    nose.tools.assert_equal(response['name'], 'actionsvc')
+    nose.tools.assert_equal(response['version'], '10.49.4-SNAPSHOT')
+
+
+@when('the action exporter endpoint info is returned')
+def action_exporter_endpoint_info_is_returned(context):
+    response = json.loads(context.response.text)
+    nose.tools.assert_equal(response['name'], 'actionexportersvc')
+    nose.tools.assert_equal(response['version'], '10.49.1-SNAPSHOT')
+
+
+@when('the backstage endpoint info is returned')
+def backstage_endpoint_info_is_returned(context):
+    response = json.loads(context.response.text)
+    nose.tools.assert_equal(response['name'], 'RAS Int Backstage Service')
+    nose.tools.assert_equal(response['version'], '0.0.1')
+
+
+@when('the case endpoint info is returned')
+def case_endpoint_info_is_returned(context):
+    response = json.loads(context.response.text)
+    nose.tools.assert_equal(response['name'], 'casesvc')
+    nose.tools.assert_equal(response['version'], '10.49.1-SNAPSHOT')
+
+
+@when('the collection exercise endpoint info is returned')
+def collection_exercise_endpoint_info_is_returned(context):
+    response = json.loads(context.response.text)
+    nose.tools.assert_equal(response['name'], 'collectionexercisesvc')
+    nose.tools.assert_equal(response['version'], '10.49.2-SNAPSHOT')
```

Comment:
We don't really want to be asserting the version of the service. In the CD world every commit will be a new version of the app and would break the tests which is not what we want. I think checking for a 200 status code would be good enough